### PR TITLE
task completed

### PR DESCRIPTION
**Pull request by Cynthia-Amanat**

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -16,47 +16,370 @@ export const quizData = {
...
+         {
+          text: 'Dutch Volcabulary',
+          href: 'https://www.learndutch.org/lessons/introduce-yourself-in-dutch/',
+        },
+        
+        
+      ],
+    },
+    {
+      text: 'What is the correct translation for "umberlla"?',
```
Comment (jontra000) [inline src/data.js]: Typo: should be `umbrella`

File: src/data.js [hunk 2]
```diff
@@ -16,47 +16,370 @@ export const quizData = {
...
+      links: [
+        {
+          text: 'Dutch Volcabulary',
+          href: 'https://www.learndutch.org/lessons/introduce-yourself-in-dutch/',
+        },
+        
+      ],
+    },
+    {
+      text: 'What is the correct translation for "mashroom"??',
```
Comment (jontra000) [inline src/data.js:141]: Typo: should be "mushroom"

Comment (Cynthia-Amanat) [reply src/data.js:141]: @jontra000 done.

File: src/data.js (hunk 1 above)
Comment (Cynthia-Amanat) [reply src/data.js]: @jontra000 done

**Pull request merged**